tests/security/fuzzy/test_fuzzy.py

```diff
@@ -290,8 +258,9 @@ def test_fuzz_procfs():
         TestRun.executor.run_expect_success('killall afl-fuzz && screen -X -S master kill')
 
     # Stop tracing
-    TestRun.executor.run_expect_success(f'kill {fio_pid}')
+    TestRun.executor.run(f'kill {fio_pid}')
```

Comment:
If You want, there's a `kill_process` method for `TestRun.executor`.

Reply:
You are right, however I need to expect success for the second 'kill' (kill_process does not check that). And I prefer to leave the first one as is to make both calls consistent.